bower.json

```diff
@@ -25,6 +26,8 @@
     "nwmatcher": "nwmatcher#1.3.4",
     "pikaday": "~1.3.2",
     "react": "#0.13.0",
-    "underscore": "underscore#1.8.2"
+    "underscore": "underscore#1.8.2",
+    "fetch": "#0.9.0",
+    "es6-promise": "~2.2.0"
```

Comment:
For now not all of our code is going through Babel, so we still need this polyfill. If we ever get to 100% Babel compiled JS, I'll keep this in mind, thanks!